src/config.rs

```diff
@@ -277,6 +341,7 @@ impl WriteCfConfig {
         cf_opts.add_table_properties_collector_factory("tikv.mvcc-properties-collector", f);
         let f = Box::new(RangePropertiesCollectorFactory::default());
         cf_opts.add_table_properties_collector_factory("tikv.range-properties-collector", f);
+        cf_opts.set_titandb_options(&self.titandb.build_opts());
```

Comment:
em, I guess we don't need to use Titan for write, lock CF, we can guarantee their data is small.

Reply:
Yes, so we turn it off by default.